Doc/library/http.server.rst

```diff
@@ -16,6 +16,14 @@
 
 This module defines classes for implementing HTTP servers (Web servers).
 
+Security Considerations
+-----------------------
+
+http.server is meant for demo purposes and does not implement the stringent
+security checks needed of real HTTP server. We do not recommend
```

Comment:
What do you think about adding an article on the following section?

`security checks needed of **a** real HTTP server`

Reply:
good point. But probably it should be either "security checks need for a real HTTP server" or "security checks need for real HTTP servers"